tests/app/views/contexts/test_section_summary_context.py

```diff
@@ -253,147 +256,149 @@ def test_context_for_section_summary_with_list_summary(companies_answer_store):
         "summary": {
             "answers_are_editable": True,
             "collapsible": False,
-            "groups": [
+            "custom_summary": None,
+            "sections": [
                 {
-                    "blocks": [],
-                    "id": "group-companies-0",
-                    "links": {},
-                    "placeholder_text": None,
-                    "title": None,
-                },
-                {
-                    "blocks": [
+                    "groups": [
                         {
-                            "add_link": "/questionnaire/companies/add-company/?return_to=section-summary",
-                            "add_link_text": "Add another UK company or branch",
-                            "empty_list_text": "No UK company or branch added",
-                            "item_anchor": "#company-or-branch-name",
-                            "item_label": "Name of UK company or branch",
-                            "list": {
-                                "editable": True,
-                                "list_items": [
-                                    {
-                                        "edit_link": "/questionnaire/companies/PlwgoG/edit-company/?return_to=section-summary",
-                                        "item_title": "company a",
-                                        "list_item_id": "PlwgoG",
-                                        "primary_person": False,
-                                        "remove_link": "/questionnaire/companies/PlwgoG/remove-company/?return_to=section-summary",
-                                    },
-                                    {
-                                        "edit_link": "/questionnaire/companies/UHPLbX/edit-company/?return_to=section-summary",
-                                        "item_title": "company b",
-                                        "list_item_id": "UHPLbX",
-                                        "primary_person": False,
-                                        "remove_link": "/questionnaire/companies/UHPLbX/remove-company/?return_to=section-summary",
+                            "blocks": [],
+                            "id": "group-companies-0",
+                            "links": {},
+                            "placeholder_text": None,
+                            "title": None,
+                        },
+                        {
+                            "blocks": [
+                                {
+                                    "add_link": "/questionnaire/companies/add-company/?return_to=section-summary",
+                                    "add_link_text": "Add another UK company or branch",
+                                    "empty_list_text": "No UK company or branch added",
+                                    "item_anchor": "#company-or-branch-name",
+                                    "item_label": "Name of UK " "company or " "branch",
```

Comment:
still a few places where this has gone wrong - I think you could fix it with a manual black command using the flag `--preview` which includes the experimental string processing option

Reply:
Used the --preview option to update these but it has converted some of them so that they span over multiple lines 😕

Reply:
I suggest use a regex to find and replace these.

Reply:
you might also be able to get away with something like
`black bad_file --preview --line-length 1000`
just to get rid of the breaks like this
then run the normal command

Reply:
Ok that worked, thanks 😀